packages/react-composites/src/composites/CallComposite/styles/ScreenSharePopup.styles.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getTheme, IButtonStyles, IModalStyleProps, IModalStyles, IStyleFunctionOrObject } from '@fluentui/react';
+import { CSSProperties } from 'react';
+
+const palette = getTheme().palette;
+
+export const screenSharePopupModalStyles: IStyleFunctionOrObject<IModalStyleProps, IModalStyles> = {
+  root: { width: '100%', height: '100%' },
+  main: {
+    boxShadow: '0px 1.2px 3.6px rgba(0, 0, 0, 0.1), 0px 6.4px 14.4px rgba(0, 0, 0, 0.13);',
+    borderRadius: '0.25rem',
```

Comment:
for borderRadius, I recommend using `theme.effects.roundedCorner`

```tsx
import { getTheme } from '@fluentui/react';
const theme = getTheme();
{
  borderRadius: theme.effects.roundedCorner4
}
```

Reply:
Thanks Anjul, changed this to use roundedCorner4.